util/dataframe_ops.py

```diff
@@ -71,9 +71,12 @@ def parse_probtest_stats(path, index_col=None):
 
 def parse_probtest_fof(path):
     ds = xr.open_dataset(path)
-    _, _, ds_veri = split_feedback_dataset(ds)
-    df_veri = ds_veri.to_dataframe().reset_index()
-    return pd.DataFrame(df_veri)
+    ds_report, ds_veri = split_feedback_dataset(ds)
```

Comment:
I think a very short doc-string for this function would be nice.
I was wondering if it would make sense to rename the variable `ds_veri` into `ds_obs`?